pallets/auction/src/lib.rs

```diff
@@ -1080,6 +1102,69 @@ pub mod pallet {
 					<ItemsInAuction<T>>::insert(item_id, true);
 					Ok(auction_id)
 				}
+				ItemId::StackableNFT(class_id, token_id, amount) => {
+					// Check available balance
+					let available_balance = T::NFTHandler::get_stackable_nft_balance(&recipient, &(class_id, token_id));
+					let reserved_balance =
+						Self::reserved_stackable_nft_balances(recipient.clone(), (class_id, token_id));
+					ensure!(
+						amount + reserved_balance <= available_balance,
```

Comment:
This needs to be safe added to prevent overflow

Reply:
should be resolved in the  following PR